autosomal_coverage.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import csv
+import argparse
+from glob import glob
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Script to calculate mean autosomal coverage after Sarek3.4.2 WGS analyses"
```

Comment:
This might become outdated if we update sarek and this script still works. Might be better to phrase it something like this: 

```suggestion
        description="Script to calculate mean autosomal coverage after WGS analyses where sarek >= v3.4.2"
```